rn/Teacher/src/modules/speedgrader/comments/ChatBubble.js

```diff
@@ -35,24 +34,26 @@ type BubbleProps = {
 
 export default class ChatBubble extends Component<BubbleProps, any> {
   render () {
-    let image
+    let tintColor
     let fromStyle
     let messageContainer
+    let transform = []
     if (this.props.from === 'me') {
-      image = images.speedGrader.chatBubbleMe
+      tintColor = colors.electric
       fromStyle = styles.myText
       messageContainer = styles.myMessageContainer
     } else {
-      image = images.speedGrader.chatBubbleThem
+      tintColor = colors.backgroundLight
       fromStyle = styles.theirText
       messageContainer = styles.theirMessageContainer
+      transform = [{ rotateY: '180deg' }]
```

Comment:
`scaleX: -1` sounds more correct here. `rotate` sounds like we'd end up with it upside down.